sync_staging.yml

```diff
@@ -0,0 +1,54 @@
+---
+resources:
+  - name: every-3-hours
+    type: time
+    source:
+      interval: 3h
+
+  - name: osbook
+    type: git
+    source:
+      uri: git@github.com:openstax/osbooks-college-algebra-bundle.git
+      branch: staging
+      private_key: ((github-private-key))
+
+jobs:
+  - name: sync-staging-branch
+    public: true
+    plan:
+    - get: every-3-hours
+      trigger: true
+    - get: osbook
+    - task: sync content
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
```

Comment:
I think we typically now try and use the [registry-image](https://github.com/concourse/registry-image-resource) resource whenever possible now in lieu of `docker-image`.